README.md

```diff
@@ -32,6 +32,14 @@ pip install .
 
 Add ``-e`` to install in [develop mode](http://pip.readthedocs.org/en/latest/reference/pip_install.html#editable-installs).
 
+``pvlib-python`` comes with Python bindings for NREL's solar position C code,
+however you must manually download the C code directly from [NREL](http://www.nrel.gov/midc/spa/).
+To use the NREL SPA algorithm,
+
+1. Download the SPA files from [NREL](http://www.nrel.gov/midc/spa/)
+2. Copy the SPA files into ``pvlib/spa_c_files``
+3. From this ``pvlib-python`` directory, run ``pip install .``
```

Comment:
On further testing, I don't actually think that my actually instructions work. 

I had to ``pip uninstall pvlib; pip install .`` to get the ``.so`` file into my ``site-packages`` directory. ``pip install . -I --no-deps`` also works. I think that the tests only passed because they were being run in the root package directory.  

I couldn't figure out the syntax to get ``python setup.py`` to do copy that file over after the package has already been installed.